### PR TITLE
README - fixes to dataproc section

### PR DESCRIPTION
**Pull request by akiezun**
fixes to dataproc section after i set it up on current dataproc

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,14 +115,15 @@ If you are looking for the codebase of the current production version of GATK, p
...
     * You must have a [Google cloud services](https://cloud.google.com/) account, and have spun up a Dataproc cluster
-      in the [Google Developer's console](https://console.developers.google.com).
+      in the [Google Developer's console](https://console.developers.google.com). You may need to have the "Allow API access to all Google Cloud services in the same project" option enabled (settable when you create a cluster).
     * You need to have installed the Google Cloud SDK from https://cloud.google.com/sdk/, since
       `gatk-launch` invokes the `gcloud` tool behind-the-scenes. As part of the installation, be sure
       that you follow the `gcloud` setup instructions [here](https://cloud.google.com/sdk/gcloud/).
     * Your inputs to the GATK need to be in Google Cloud Storage buckets, and should be specified on
       your GATK command line using the syntax `gs://my-gcs-bucket/path/to/my-file`
+   * You may need to pass your credentials explicitly, e.g., to pass the API key use the `--apiKey` argument to GATK (you can create an API key on the Credentials tab of the API Manager page, i.e., `https://console.cloud.google.com/apis/credentials?project=<your_gcloud_project>`
```
Comment (droazen) [inline README.md]: Missing closing parenthesis.

Comment (droazen) [inline README.md]: Also, this URI is likely to get stale in the future -- recommend saying just "you can create an API key on the Credentials tab of the API Manager page"

File: README.md [hunk 2]
```diff
@@ -146,18 +148,20 @@ If you are looking for the codebase of the current production version of GATK, p
             --num-executors 5 --executor-cores 2 --executor-memory 4g \
             --conf spark.yarn.executor.memoryOverhead=600
         ```
-
+    * When using Dataproc you can access Web interface from YARN, Hadoop and HDFS. Follow [those instructions] (https://cloud.google.com/dataproc/cluster-web-interfaces) to create an SSH tunnel and connect with your browser.
```
Comment (droazen) [inline README.md]: "access Web interface from" -> "access the web interfaces for"
"those instructions" -> "these instructions"

File: README.md [hunk 3]
```diff
@@ -146,18 +148,20 @@ If you are looking for the codebase of the current production version of GATK, p
...
     * Note that the spark-specific arguments are separated from the tool-specific arguments by a `--`.
     * If you want to avoid uploading the GATK jar to GCS on every run, set the `GATK_GCS_STAGING`
       environment variable to a bucket you have write access to (eg., `export GATK_GCS_STAGING=gs://<my_bucket>/`)
     * For further instructions on using GATK with Google Cloud Dataproc, see our
       [Cloud Dataproc wiki page](https://github.com/broadinstitute/gatk/wiki/Running-GATK-on-Cloud-Dataproc)
```
Comment (droazen) [inline README.md]: Is this wiki page out-of-date now?

**Comment by droazen** [comment]
back to @akiezun, merge after addressing comments

File: README.md (hunk 1 above)
Comment (akiezun) [inline README.md]: fixed

File: README.md (hunk 2 above)
Comment (akiezun) [inline README.md]: fixed

File: README.md (hunk 3 above)
Comment (akiezun) [inline README.md]: yes, the wiki should be deleted i think

Comment (droazen) [inline README.md]: Can you delete it then?

Comment (akiezun) [inline README.md]: done

**Pull request merged**